pkg/template/diff_printer.go

```diff
@@ -0,0 +1,180 @@
+package template
+
+import (
+	"fmt"
+	"io"
+	"strconv"
+	"strings"
+
+	"github.com/olekukonko/tablewriter"
+)
+
+type DiffPrinter struct {
+	w      io.Writer
```

Comment:
For better separation of concerns, I'm considering:
1. Making this a `bytes.Buffer`
2. Removing the `io.Writer` arg to `NewDiffPrinter`
3. Having `Render()` return a string

What do you think?

Reply:
I do associate the term "Render" with actually printing/outputting something to the screen, so having it return a string would be a little less intuitive to me at first glance. I can see what you mean about separation of concerns though...right now `Render()` prepares the output and prints it (or writes it to `w`) both. I'm used to that though, since that's on-par with how, for example, `render()` in a React component works, more or less.

Reply:
Hmmm ok, in that case I'll leave it as-is :+1: